cmd/toolbox.lima

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
This is quite different from other `*.lima` wrappers, as this doesn't even execute `limactl`

Reply:
I also don't like the name of the "toolbox", as it is likely to conflict with something else, e.g., https://android.googlesource.com/platform/system/core/+/froyo/toolbox/Android.mk

Reply:
https://github.com/containerd/nerdctl/tree/main/examples might be a better place?

Reply:
The script executes itself with `lima` (if needed), so it should use limactl (eventually)
As mention in the issue, the "toolbox" is somewhat overloaded with 3+ implementations